.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+### Type of change
+
+- [ ] bug fix
+- [ ] new feature
+- [ ] documentation
+- [ ] other (describe)
+
+### Scope of changes
+
+Briefly describe the scope of changes. Include links to ShortCut or Jira ticket if applicable.
+
+### Acceptance criteria
+
+Describe how reviewers can test this change to be sure that it works correctly. Add a checklist if possible.
+
+### Author checklist
+
+- [ ] I have manually tested the change and/or added automation in the form of unit tests or integration tests
+- [ ] I have added migrations for any changes to the database
+- [ ] My migrations can be run before the code is deployed without causing any issues
+- [ ] Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
```

Comment:
Let's remove this one since our current protocol is that everyone merges their own PRs after they have been approved